pageserver/src/tenant.rs

```diff
@@ -7195,7 +7195,7 @@ mod tests {
             let end = desc
                 .key_range
                 .start
-                .add(Timeline::MAX_GET_VECTORED_KEYS.try_into().unwrap());
```

Comment:
I don't see `MAX_GET_VECTORED_KEYS` being removed. Let's remove it altogether since it's confusing now.
You'll have to update the tests that use it.

Reply:
You mean the constant `MAX_GET_VECTORED_KEYS`? I missed a place in `test_read_path` since the feature is not enabled in my IDE. All usage of `MAX_GET_VECTORED_KEYS` should be gone now :)

Reply:
@VladLazar what do you think about the latest commit?

Reply:
Thanks for the update. lgtm - I've kicked a ci run